frontend/src/views/main/project/FiltersCatalog.vue

```diff
@@ -236,12 +232,15 @@ async function runSlicingFunction() {
         })
     }
 
+    mixpanel.track("Run slicing function from Catalog", {
+        slicingFunctionName: selected.value!.name,
```

Comment:
Maybe we can add datasetId and anonymized input parameters.

Reply:
How can I get the inputs in anonymized way?